packages/editor/src/components/block-drop-zone/style.scss

```diff
@@ -8,14 +8,12 @@
 	}
 
 	&.is-close-to-bottom {
-		background: none;
```

Comment:
If we remove this, then it will bring back #6035, where blocks change color when dragging another block over them.